docs/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+Closes #
+
+Changes proposed in this pull request:
+
+-
+-
+-
+
+Pull request checklist:
+
+- [ ] branch is based on `master`
```

Comment:
```suggestion
```
You have to go through a lot of effort to do anything else; this seems like useless noise to me.

Reply:
And also considering that GitHub warns if the branch is not based on `master`, we can remove this point indeed :+1: